Formula/crystal.rb

```diff
@@ -95,12 +110,12 @@ def install
     # Setup the same path the wrapper script would, but just for building shards.
     # NOTE: it seems that the installed crystal in bin/"crystal" can be used while
     #       building the formula. Otherwise this ad-hoc setup could be avoided.
-    embedded_crystal_path=`"#{buildpath/".build/crystal"}" env CRYSTAL_PATH`.strip
-    ENV["CRYSTAL_PATH"] = "#{embedded_crystal_path}:#{buildpath/"src"}"
+    embedded_crystal_path = Utils.safe_popen_read(buildpath/".build/crystal", "env", "CRYSTAL_PATH").strip
+    ENV["CRYSTAL_PATH"] = "#{embedded_crystal_path}:#{buildpath}/src"
```

Comment:
This configuration should be unnecessary because we're using the source tree wrapper script which sets `CRYSTAL_PATH` to the correct stdlib source location.